src/main/java/uk/gov/ida/matchingserviceadapter/configuration/EuropeanConfigurationDefaultsHelper.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ida.matchingserviceadapter.configuration;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.eclipse.jetty.util.TypeUtil.asList;
+
+public class EuropeanConfigurationDefaultsHelper {
+
+    private static final String[] ACCEPTABLE_HUB_CONNECTOR_IDS_PRODUCTION = new String[] {
```

Comment:
```suggestion
    private static final String[] ACCEPTABLE_HUB_CONNECTOR_ENTITY_IDS_PRODUCTION = new String[] {
```
`entity ID` is a specific SAML domain name - let's use it

Reply:
👍